README.md

```diff
@@ -212,6 +212,31 @@ async function main() {
 main();
 ```
 
+### Generate Content
+
+#### How to structure `contents` argument for `generateContent`
+
+The SDK allows you to specify the following types in the `contents` parameter:
+
+#### Content
+
+- `Content`: The SDK will wrap the singular `Content` instance in an array which
```

Comment:
As a user, looking at the README, I don't think I care about how the SDK internally handles my type. If I'm here I just probably want to know how to use it, I'd say we just show a few samples of what's allowed.